docs/index.md

```diff
@@ -17,6 +17,7 @@ Prettier is an opinionated code formatter with support for:
 - [GraphQL](http://graphql.org/)
 - [Markdown](http://commonmark.org/), including [GFM](https://github.github.com/gfm/) and [MDX](https://mdxjs.com/)
 - [YAML](http://yaml.org/)
+- [MJML](https://github.com/mjmlio/mjml)
```

Comment:
I don’t think we should add it to the docs explicitly, it’s just the same as the HTML formatting, we don’t need to add every file extension to the support list.